Adding markdown-table-editor plugin

ganesshkumar opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

Link to my plugin: https://github.com/ganesshkumar/obsidian-table-editor

## Release Checklist
- [x] I have tested the plugin on
  - [x]  Windows
  - [ ]  macOS
  - [ ]  Linux
  - [x]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the tips in https://github.com/obsidianmd/obsidian-releases/blob/master/plugin-review.md and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

lishid commented:
https://github.com/ganesshkumar/obsidian-table-editor/blob/fcb1ab7fd735f446a288cfeb426d802c18cee333/src/main.ts#L4
Please use a less generic name here :)

Otherwise, yeah what Liam said!

ganesshkumar commented:
Thank you @liamcain and @lishid for your comments.

I have updated the codebase and released a new version [0.1.1](https://github.com/ganesshkumar/obsidian-table-editor/releases/tag/0.1.1) 

1. @liamcain, `papaparse` was able to auto detect the delimiter and handle markdown, CSV and Excel data formats. I am using it to parse any inputs.
2. @lishid , my bad to miss that. I have updated the name of the plugin class.
